demonstrations/tutorial_measurement_optimize.py

```diff
@@ -0,0 +1,830 @@
+r"""
+Measurement optimization
+========================
+
+.. meta::
+    :property="og:description": Optimize and reduce the number of measurements required to evaluate a variational algorithm cost function.
+    :property="og:image": https://pennylane.ai/qml/_images/grouping.png
+
+.. related::
+
+   tutorial_vqe Variational quantum eigensolver
+   tutorial_quantum_chemistry Quantum chemistry with PennyLane
+   tutorial_qaoa_intro Intro to QAOA
+
+The variational quantum eigensolver (VQE) is the OG variational quantum algorithm. Harnessing
+near-term quantum hardware to solve for the electronic structure of molecules, VQE is *the*
+algorithm that sparked the variational circuit craze of the last 5 years, and holds the greatest
+promise for showcasing a quantum advantage on near-term quantum hardware. It has also inspired
+other quantum algorithms such as the :doc:`Quantum Approximate Optimization Algorithm (QAOA)
+</demos/tutorial_qaoa_intro>`.
+
+To scale VQE beyond the regime of classical computation, however, we need to use it to solve for the

```

Comment:
```suggestion
To scale VQE beyond the regime of classical computation, however, we need to solve for the

```
I feel like "to use it" can be assumed and removed to make a more concise sentence.